sql-plugin/src/main/scala/com/nvidia/spark/rapids/GpuOverrides.scala

```diff
@@ -2183,13 +2183,18 @@ object GpuOverrides {
         ("key", TypeSig.commonCudfTypes, TypeSig.all)),
       (in, conf, p, r) => new BinaryExprMeta[ArrayContains](in, conf, p, r) {
         override def tagExprForGpu(): Unit = {
+          // avoid using BinaryExpression accessors since hierarchy changed in Spark 3.2
```

Comment:
wonder if we want an explicit comment saying  left and right must match order in children